Information about Huawei devices (#4289)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/android/android-faqs.md

Yes! You can use Segment's browserify'd [analytics-node](https://github.com/segmentio/analytics-node) package just like any other client-side JavaScript library.

## Are there any limitations for using Segment with Huawei?

> No. Currently we haven't had any instances that shows there are limitations when Segment is tracking Huawei devices.

Reviewer: ```suggestion
No, there hasn't been any instances that show there are limitations when Segment tracks Huawei devices.
```
